### DISCUSSION TITLE
关于GPU版本

### DISCUSSION THREAD

**Discussion by steinaffe** [YaoFANGUK/video-subtitle-extractor#157] [Q&A] [answered]
请问，GPU版本是否支持intel显卡硬件呢？

**Comment by YaoFANGUK** [comment, marked as answer]
intel的是核显吧，只能跑CPU，这里说的GPU指的是AMD或者Nvidia的独立显卡，当然AMD不支持，因为AMD的显卡不能使用CUDA计算库，所以只支持Nvidia的显卡
